Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -123,6 +157,8 @@ public class MapViewModel : BaseViewModel {
                 _webSocketService.ResumeGame += ResumeGame;
                 _webSocketService.PauseGame += PauseGame;
                 _webSocketService.EndGame += EndGame;
+
+                _webSocketService.IntervalEvent += Test;
```

Comment:
Is deze nog nodig?

Reply:
Mijn taken betreft is het alleen om te testen of de intervals binnen komen. Maar Jesse kan wat ik verder in die functie heb gezet gebruiken voor het updaten en weergeven van spelers.